src/formatter.js

```diff
@@ -216,9 +141,7 @@ export class Formatter {
     const beams = options.auto_beam ? Beam.applyAndGetBeams(voice) : [];
 
     // Instantiate a `Formatter` and format the notes.
-    new Formatter()
-      .joinVoices([voice])
-      .formatToStave([voice], stave, { align_rests: options.align_rests, stave });
+    new Formatter().joinVoices([voice]).formatToStave([voice], stave, { align_rests: options.align_rests, stave });
```

Comment:
I'm -1 on refactors that put things into a single long line that previously showed each of the steps involved.  If a linter doesn't like having `.dotNotation` as a line continuation, better to me is to create a temporary variable.